circuits/cpp/barretenberg/scripts/decls_json.py

```diff
@@ -36,7 +36,7 @@ def process_files(files: List[str]) -> List[dict]:
             '-isystem', '/usr/include/c++/10',
```

Comment:
This file is never tested in CI, so CI won't show any failures if it was not working regardless